main.go

```diff
@@ -59,12 +59,19 @@ func main() {
 		KeyFilePath:         keyFilePath,
 	}
 
+	eventDispatcher := dep.InitEventDispatcher()
+
+	defer func() {
```

Comment:
`InitEventDispatcher` should be renamed to `InjectEventDispatcher` for consistency. Dispatcher should be closed inside execute function.

Reply:
Regarding renaming:
* Should we rename https://github.com/short-d/kgs/blob/master/dep/wire.go#L40?

I don't see any reason to rename it. I mean, this function just initializes an `event dispatcher`

Reply:
@alldroll sorry. I think I did the renaming for Short😅

Reply:
Let’s keep `init` here!